go/mysqlconn/replication/constants.go

```diff
@@ -0,0 +1,91 @@
+package replication
+
+// This file contains the constant definitions for this package.
+
+// This is the data type for a field.
+// Values taken from include/mysql/mysql_com.h
+// FIXME(alainjobart) this is the same info as constants describes in
```

Comment:
sqltypes.Value also has some duplication that will require consolidation.
I also think the ones in mysql.go are unused now.